report/data_tasks/report/create_from_scratch/02_hubspot_read/02_data_import/01_hubspot_import.py

```diff
@@ -102,3 +103,17 @@ def main(connection, **kwargs):
         items=company_deals,
         fields=COMPANY_DEAL_FIELDS,
     )
+
+
+def _sort_deal_dates(deals):
+    """Ensure the start of a deal comes before the end.
+
+    This should be true, but as these values are entered by people there's
+    nothing really stopping them from being put in backwards, which will cause
+    Postgres conniptions
+    """
+    for deal in deals:
+        services_start, services_end = deal["services_start"], deal["services_end"]
+        if services_start and services_end:
```

Comment:
Would services_start == services_end cause issues?

Reply:
I don't think so, as the max and min are the same, so which equal thing you get wouldn't matter?